doc/MONSTERS.md

```diff
@@ -24,25 +26,44 @@ Monster types support the following properties (mandatory, except if noted other
 "name": { "ctxt": "fish", "str": "pike", "str_pl": "pikes" }
 ```
 
-Name displayed in-game, and optionally the plural name and a translation context.
+Name displayed in-game, and optionally the plural name and a translation context (ctxt).
```

Comment:
This makes it seem as thought ctxt is mandatory, when it is not, and only needs to be used in ambiguous cases.

Reply:
I agree, but note that the original had the same issue.

Reply:
Actually, no it didn't nor does this one - I missed the optionally bit.